chrome_extension/background.js

```diff
@@ -0,0 +1,49 @@
+/**
+ * @file background.js
+ * @description Creates a new background script that listens for messages from the content script
+ * and sends them to a WebSocket server.
+*/
+
+let socket;
+let TIMEOUT_INTERVAL = 5000; // ms
```

Comment:
What do you think about renaming this to `RECONNECT_TIMEOUT_INTERVAL`? Also, why 5s instead of 1s?